server/routes/place/html.py

```diff
@@ -71,11 +71,18 @@ def place(place_dcid=None):
     place_name = place_names[place_dcid]
   else:
     place_name = place_dcid
+
+  place_summary = None
+  if not category:
+    # Only show summary for Overview
+    place_summary = current_app.config['PLACE_EXPLORER_SUMMARIES'].get(place_dcid, "")
```

Comment:
Should we put fetching (and showing) the summary behind a request flag initially?

Reply:
i have it enabled only for local and autopush - sorry that didn't get pushed!